comps/chathistory/mongo/requirements.txt

```diff
@@ -1 +1 @@
-motor==3.4.0
+motor
```

Comment:
does something wrong with the version?

Reply:
yes, motor==3.4.0 does not match the latest mongo version.
need to update motor version to fix `ImportError: cannot import name '_QUERY_OPTIONS' from 'pymongo.cursor' (/home/user/.local/lib/python3.11/site-packages/pymongo/cursor.py)` issue